lib/ansible/modules/network/infinity/infinity.py

```diff
@@ -0,0 +1,650 @@
+# !/usr/bin/python
+# meiliu@fusionlayer.com
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+
+has_url_parse = True
+try:
+    # python3
+    import urllib.request as urllib_request
+    import urllib.parse as urllib_parse
+    from urllib.request import AbstractHTTPHandler
+    has_url_parse = True
+except ImportError as e:
+    # python2
+    has_url_parse = False
+    # print(" using python2 librarary")
+    import urllib2 as urllib2_request
+    import urllib
+    import base64
+
+from ansible.module_utils.basic import *
```

Comment:
Don't import `*`, import only what you need

See: http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#python-imports